README.md

```diff
@@ -59,11 +59,11 @@ search these unofficial resources:
 * [Node.js Slack Community](https://node-js.slack.com/): Visit
   [nodeslackers.com](http://www.nodeslackers.com/) to register.
 
-GitHub issues are meant for tracking enhancements and bugs, not general support.
+GitHub issues are for tracking enhancements and bugs, not general support.
 
-Remember, libre != gratis; the open source license grants you the freedom to use
-and modify, but not commitments of other people's time. Please be respectful,
-and set your expectations accordingly.
+The open source license grants you the freedom to use Node.js. It does not
```

Comment:
I think the `and modify` bit here could stay?

Reply:
My thinking was that the MIT license says all of this:

> use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies 

I understand why we'd select _use_ from that list, but it's less clear why we'd also include _modify_ over _copy_ or _distribute_. Just sticking with _use_ seemed simplest and best.

So that was my thinking. If that's not persuasive, I'll restore _and modify_ to the text.

Reply:
@Trott That’s okay with me, yeah… I guess I’ve thought that `modify` would be the least likely of the verbs you used for which it would be assumed that it’s okay to do?